src/main/java/uk/gov/hmcts/reform/em/stitching/domain/Bundle.java

```diff
@@ -225,4 +231,39 @@ public String toString() {
                 + this.hasTableOfContents + ", hasCoversheets=" + this.hasCoversheets + ", hasFolderCoversheets="
                 + this.hasFolderCoversheets + ")";
     }
+
+    @Transient
+    public Integer getSubtiles(SortableBundleItem container, Map<BundleDocument, File> documents) {
+        return container
+                .getSortedItems().flatMap(SortableBundleItem::getSortedDocuments)
+                .map(i -> {
+                    try {
+                        return load(documents.get(i)).getDocumentCatalog().getDocumentOutline();
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+                    return null;
+                })
+               .filter(o->o!=null)
+               .filter(o->o.getFirstChild()!=null)
```

Comment:
maybe combine into a single one:

```
.filter(o -> o != null && o.getFirstChild() != null)
```

Reply:
ok Linus , i did likewise before , it was failing to filter properly so i splitted. Will try again .Thanks

Reply:
made this change and its working fine. Thanks